### PR TITLE
Added a function to disconnect from server

### PR DESCRIPTION
**Pull request by maxpaynestory**
Added the functionality to disconnect from socket.io or pusher servers. This feature was requested on #94

### CODE REVIEW

File: src/echo.ts [hunk 1]
```diff
@@ -10,9 +10,9 @@ class Echo {
     /**
      * The broadcasting connector.
      *
-     * @type {object}
+     * @type {Connector}
      */
-    connector: any;
+    connector: Connector;
```
Comment (taylorotwell) [inline src/echo.ts]: This change seems irrelevant? Please revert this and the above change.

Comment (maxpaynestory) [inline src/echo.ts]: I have reverted the irrelevant part.
